modules/ROOT/pages/amazon/amazon-ec2-connector.adoc

```diff
@@ -327,49 +474,54 @@ amazon.secretkey=<Secret Key>
 amazon.region=<Region>
 ----
 +
-. Drag an HTTP Listener component onto the canvas and configure the following parameters:
+. Drag an HTTP Listener operation onto the canvas and configure the following parameters:
 +
 image::amazon/amazon-ec2-http-props.png[ec2 HTTP configure properties]
 +
-[%header%autowidth.spread]
+[%header,cols="30s,70a"]
 |===
-|Parameter |Value
+|Studio Field |Value
 |Display Name |Listener
 |Extension Configuration | If no HTTP Listener configuration has been created yet, click the plus sign to add a new HTTP Listener Configuration and click OK (Give values "localhost" and "8081" for Host and Port columns).
 |Path |/createVolume
 |===
 +
-. Drag the Amazon EC2 Connector "Create volume" next to the HTTP Listener component.
-. Configure the EC2 connector by adding a new Amazon EC2 Global Element. Click the plus sign next to the Connector Configuration field.
+. Drag the Amazon EC2 Connector *Create volume* operation next to the HTTP Listener component.
+. Configure the EC2 connector by adding a new Amazon EC2 global element. Click *+* next to the *Connector Configuration* field.
 .. Configure the global element according to the table below:
 +
-[%header%autowidth.spread]
+[%header,cols="30s,40a,30a"]
 |===
-|Parameter |Description |Value
-|Name |Enter a name for the configuration to reference it. |<Configuration_Name>
+|Studio Field |Description |Value
+|Name |Enter a name for the configuration to reference it. |`<Configuration_Name>`
 |Access Key |Alphanumeric text string that uniquely identifies the user who owns the account. |`${amazon.accesskey}`
 |Secret Key |Key that plays the role of a password. |`${amazon.secretkey}`
-|Region Endpoint |Region to be select from drop down for the Amazon EC2 Client. |USEAST1
+|Region Endpoint |Region to be select from drop down for the Amazon EC2 Client. |`USEAST1`
 |===
 .. Your configuration should look like this:
 +
 image::amazon/amazon-ec2-use-case-config.png[ec2 use case config]
 +
 .. The corresponding XML configuration should be as follows:
 +
-[source,xml]
+[source,xml,linenums]
 ----
-<ec2:config name="Amazon_EC2_configuration" doc:name="Amazon EC2 Configuration" >
-<ec2:basic-connection accessKey="${amazon.accesskey}" secretKey="${amazon.secretkey}" region="${amazon.region}" />
+<ec2:config 
+	name="Amazon_EC2_configuration" 
+	doc:name="Amazon EC2 Configuration" >
+<ec2:basic-connection 
+	accessKey="${amazon.accesskey}" 
+	secretKey="${amazon.secretkey}" 
+	region="${amazon.region}" />
 </ec2:config>
 ----
 +
-. Click Test Connection to confirm that Mule can connect with the EC2 instance. If the connection is successful, click OK to save the configurations. Otherwise, review or correct any incorrect parameters, then test again.
+. Click *Test Connection* to confirm that Mule can connect with the EC2 instance. If the connection is successful, click *OK* to save the configurations. Otherwise, review or correct any incorrect parameters, then test again.
```

Comment:
```suggestion
. Click *Test Connection*.
. If the connection is successful, click *OK* to save the configuration; otherwise, review or correct any incorrect parameters, and then test again.
```